lease-kubernetes/src/main/resources/reference.conf

```diff
@@ -0,0 +1,52 @@
+akka.coordination.lease.kubernetes {
+
+    lease-class = "akka.coordination.lease.kubernetes.KubernetesLease"
+
+    api-ca-path = "/var/run/secrets/kubernetes.io/serviceaccount/ca.crt"
+    api-token-path = "/var/run/secrets/kubernetes.io/serviceaccount/token"
+
+    # Host for the Kubernetes API server. Typically this will be set via an environment
+    # variable that is set when running inside Kubernetes
+    api-service-host = "localhost"
+    api-service-host = ${?KUBERNETES_SERVICE_HOST}
+
+    # Port for the Kubernetes API server. Typically this will be set via an environment
+    # variable that is set when running inside Kubernetes
+    api-service-port = 8080
+    api-service-port = ${?KUBERNETES_SERVICE_PORT}
+
+    # Namespace file path. The namespace is to create the lock in. Can be overridden by "namespace"
+    #
+    # If this path doesn't exist, the namespace will default to "default".
+    namespace-path = "/var/run/secrets/kubernetes.io/serviceaccount/namespace"
+
+    # Namespace to create the lock in. If set to something other than "<namespace>" then overrides any value
+    # in "namespace-path"
+    namespace = "<namespace>"
+
+    # How often to write time into CRD so that if the holder crashes
+    # another node can take the lease after a given timeout. If left blank then the default is
+    # max(5s, heartbeat-timeout / 10)
+    heartbeat-interval = ""
+    #heartbeat-interval = 12s
```

Comment:
A bit confusing, perhaps just mention that the default will end up being 12s in the comment instead?

Reply:
👍